include/internal/namemap.h

```diff
@@ -16,6 +16,6 @@ OSSL_NAMEMAP *ossl_namemap_stored(OPENSSL_CTX *libctx);
 OSSL_NAMEMAP *ossl_namemap_new(void);
 void ossl_namemap_free(OSSL_NAMEMAP *namemap);
 
-int ossl_namemap_add(OSSL_NAMEMAP *namemap, const char *name);
-const char *ossl_namemap_name(const OSSL_NAMEMAP *namemap, int number);
+int ossl_namemap_add(OSSL_NAMEMAP *namemap, int number, const char *name);
+const char **ossl_namemap_names(const OSSL_NAMEMAP *namemap, int number);
```

Comment:
Alternate names?
ossl_namemap_id2names
ossl_namemap_name2id

Reply:
That's a though...  although, I would prefer 'num' over 'id'